packages/react/src/components/Tile/tile-story.scss

```diff
@@ -4,6 +4,6 @@
   @include tile.tile($enable-experimental-tile-contrast: true);
 }
 
-div .bx--tile--selectable:not(:last-child) {
+div .cds--tile--selectable:not(:last-child) {
   margin-block-end: 1rem;
 }
```

Comment:
Just confirming we want this visual change to the Tile stories?

<img width="1254" alt="Screenshot 2023-11-17 at 12 27 01 PM" src="https://github.com/carbon-design-system/carbon/assets/2753488/91c35eea-975d-423a-b0df-cb1659ca227b">

Reply:
Just matching the styles we had in the v10 storybook, must have been missed in the migration: https://v7-react.carbondesignsystem.com/?path=/story/components-tile--multi-select